Example/FirestoreSample/FirestoreSample/Views/FavouriteFruitsAsyncSequenceView.swift

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+
+private struct Fruit: Codable, Identifiable, Equatable {
+  @DocumentID var id: String?
+  var name: String
+  var isFavourite: Bool
+}
+
+struct FavouriteFruitsAsyncSequenceView: View {
+  @State fileprivate var fruits: [Fruit] = []
+
+  @State var showOnlyFavourites = true
+
+  var body: some View {
+    List(fruits) { fruit in
+      Text(fruit.name)
+    }
+    .task {
+      do {
+        let collection = Firestore.firestore().collection("fruits")
+        for try await fruits in collection.addSnapshotListener(Fruit.self) {
```

Comment:
Should it be `snapshotSequence` instead of `addSnapshotListener`?

Reply:
Good idea - I made this change in [5a68fdf](https://github.com/firebase/firebase-ios-sdk/pull/11508/commits/5a68fdff8d4c3c360185fd4179efbf5c1ff72f6d) - WDYT?

Reply:
LGTM. Thanks